tinygrad/tensor.py

```diff
@@ -2688,6 +2688,19 @@ def celu(self, alpha=1.0):
     """
     return self.maximum(0) + (alpha * ((self / alpha).exp() - 1)).minimum(0)
 
+  def selu(self, alpha=1.67326, scale=1.0507):
```

Comment:
just use alpha and gamma? it matches onnx naming and one less indirection

Reply:
<img width="839" alt="image" src="https://github.com/user-attachments/assets/26e7660f-c1d3-4870-a1b8-4721feccfc99">

pytorch called it `scale` but I guess people just call `.selu()` anyways 
will change to gamma